### DISCUSSION TITLE
Move song progress and unstable rate settings back to settings sidebar

### DISCUSSION THREAD

**Discussion by Anomalyhun** [ppy/osu#18212] [Ideas]
Please make that the progressbar can be disabled outside the skin editor in the setting menu and that the UR bar snap down in the middle and the combo numbers when you disable it.

**Comment by Anomalyhun** [comment]
And the UR bar size can be changed in the setting menu too.

**Comment by bdach** [comment]
This change was intentional. You can read about some of the reasons why [in this thread](https://github.com/ppy/osu/pull/17586), but to sum up:

* The settings sidebar has too many options in it and nobody understands what half of them do. When skin-related settings are moved to the skin editor, you instantly see feedback about what the setting does.
* Having multiple layers of settings can cause confusion because you'll change a setting on a skin and then it won't do anything because you have the setting overridden on a user level.

That said this wasn't final. Let's see how much support this discussion gets and we'll figure out whether this really needs changing from there.
